server/api.js

```diff
@@ -7,51 +7,78 @@ const itemsPerPage = 25;
 const router = Router();
 
 const allowlist = {
-	POST: ["/sign-in"],
+	POST: {
+		"/sign-in": "public",
+		"/tender": "token",
+	},
+	GET: {
+		"/skills": "token",
+		"/buyer-tender": "token",
+		"/bidder-bid": "token",
+		"/tenders": "token",
+	},
 };
 
 const auth = async (req, res, next) => {
 	try {
 		const method = req.method.toUpperCase();
 		const path = req.path;
 
-		if (allowlist[method] && allowlist[method].includes(path)) {
+		const allowedAccess = allowlist[method] && allowlist[method][path];
+
+		if (allowedAccess === "public") {
 			return next();
 		}
 
-		const authHeader = req.headers.authorization;
-		const token = authHeader?.split(" ")[1];
+		if (allowedAccess === "token") {
```

Comment:
Admins should have access to everything even if it's not listed in the allowlist